R/do_bayes_ab.R

```diff
@@ -236,26 +236,30 @@ tidy.bayesTest <- function(x, percentLift = 0, credInt = 0.9, type = "summary",
     beta_b <- density(probability_b, n = 2048)
     indice_b <- beta_b$x > 0 & beta_b$x < 1
 
-    ab_identifier <- if(!is.null(x$ab_identifier) && length(x$ab_identifier) == 2){
-      x$ab_identifier
-    } else {
-      c("A", "B")
-    }
-
     a_data <- data.frame(
-      ab_identifier = ab_identifier[[1]],
+      ab_identifier = rep("A", length(indice_a)),
       conversion_rate_pct = beta_a$x[indice_a] * 100,
       probability_density = beta_a$y[indice_a],
       stringsAsFactors = FALSE
     )
 
     b_data <- data.frame(
-      ab_identifier = ab_identifier[[2]],
+      ab_identifier = rep("B", length(indice_a)),
```

Comment:
Ah, right, it automatically replicate single value, so no need for rep. Will remove it